ci/build.yml

```diff
@@ -2,6 +2,38 @@
 # SPDX-License-Identifier: Apache-2.0
 
 jobs:
+
+- job: fix_bazel_cache
+  timeoutInMinutes: 120
+  pool:
+    name: 'ubuntu_20_04'
+    demands: assignment -equals default
+  steps:
+    - checkout: self
+    - bash: ci/dev-env-install.sh
+      displayName: 'Build/Install the Developer Environment'
+    - template: ci/bash-lib.yml
+      parameters:
+        var_name: bash-lib
+    - bash: |
+        set -euo pipefail
+        eval "$(dev-env/bin/dade assist)"
+
+        bazel build //ci/cron:cron
+
+        key=$(mktemp)
+        cleanup="rm -rf $key ~/.config/gcloud"
+        trap "$cleanup" EXIT
+        echo "$GCRED" > $key
+        gcloud auth activate-service-account --key-file=$key
```

Comment:
do we have delete permissions on PRs?

Reply:
Good question, let me check... Yes, we do. Bazel needs it in general to manage the cache.